{
  "title": "query column does not exist in dataset -> should reject with InsightError",
  "input": {
    "WHERE": {
      "GT": {
        "sections_avg": 97
      }
    },
    "OPTIONS": {
      "COLUMNS": [
        "sections_room",
        "sections_avg"
      ],
      "ORDER": "sections_avg"
    }
  },
  "errorExpected": true,
  "expected": "InsightError"
}
